README.md

```diff
@@ -1 +1,11 @@
 # hw01
+# Homework assignment
+## Emma Gilheany
+### Week 1 Comp for Social Science
+
+I am a 5th year PhD student in Anthropology who has **absolutely no** programming experience. My dissertation is a *qualitative and creative* humanistic inquiry into the recent history of Hopedale, Nunatsiavut. Here is a [somewhat outdated summary](https://cissr.uchicago.edu/research/2018-2019/field-research/emma-gilheany) of what I am working on. Lately, I am fascinated by
+* Cold War bases in the Arctic built by Americans and the USSR to surveil for missiles
+* US experiments with climate and *recreating* Arctic climates for combat-training purposes
+* Why Americans view Canada as a place with somehow better politics than the US, **despite** glaring evidence to the contrary, particularly concerning Canada's treatment of indigenous people
+* Here is a picture of what the Hopedale radar base looked like in the 1960s ![Image of radar base](RCAF_Station_Hopedale_-_1964-1965.jpg)
+* And here is a picture taken in 2014 ![Image of Hopedale](Hopedale-Nunatsiavut.jpg)
```

Comment:
Good use of images, but in the document the position of the image looks a little bit weird, maybe consider moving the last image to another line, instead of putting it right after the text. But that is the only advice that I can give. Overall, great work!